### DISCUSSION TITLE
109

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#32] [General]
# 潮流周刊第109期 - 新疆奶茶

封面来源于周末吃的一家新疆菜「冒烟的阿凡提」里面的咸奶茶，还点了手抓饭、烤包子、大盘鸡、串都很正宗很好吃，很值得一去。

https://weekly.tw93.fun/posts/109-%E6%96%B0%E7%96%86%E5%A5%B6%E8%8C%B6

**Comment by tw93** [comment]
话说 weekly 的这个字体和之前版本对比起来，之前的更细一些，你更喜欢老的还是新的？

**Comment by Du0yu** [comment]
这个字体好好看，我想问一下这是什么字体呢？

**Reply by tw93** [reply]
https://github.com/lxgw/LxgwWenKai
可以试试，我用的这个得阅读器的版本，之前使用的正常版本，放到这里容易显得有些细

**Reply by Du0yu** [reply]
Thank you so much!

**Comment by HeroadZ** [comment]
2022 年 CSS 生态调查那个网站打开需要密码了。有人知道密码是什么吗

**Reply by tw93** [reply]
看起来是官方有数据要修改，暂时关了导致，可见这里 https://stateofcss.com/en-us/
假如需要看，可以看这个 https://www.51cto.com/article/741702.html

**Comment by tumars** [comment]
图解浏览器的多进程渲染机制，这篇文章是抄袭啊，内容和配图都是使用李兵的浏览器工作原理与实践
